ce/marketing/developer/event-management-web-application.md

```diff
@@ -18,88 +18,114 @@ manager: kvivek
 
-The Dynamics 365 Event Management feature helps you to organize, manage and process the events. When you install the Event Management solution, you will get an **Event Portal** which gives the users a web application where they can know more about the event details. More information: [Event Portal](https://docs.microsoft.com/en-us/dynamics365/customer-engagement/marketing/set-up-event-portal).
-
-
-The Event Management web application consists of two parts:
+The Event Management event portal consists of two parts:
 - Angular frontend
 - REST API backend
 
-The frontend part is a single page application which is created using [Angular](https://angular.io) framework. The frontend part of the web application is fully customizable. Backend and customizability of the backend depends on hosting type.
+The frontend part is a single page application which is created using the [Angular](https://angular.io) framework. The frontend part of the web application is fully customizable. The backend and the customizability of the backend depend on the hosting type.
 
- You can host the Event Management web application using
+There are two hosting types for the event portal:
 - [Dynamics 365 Portal hosted](#dynamics-365-portal-hosted)
-<!--- [Self-hosted](#self-hosted)-->
-
-## Dynamics 365 Portal hosted
-Event Management web application comes as a Dynamics 365 Portal hosted web when you install the Event Management solution.
-
-The frontend part of the application is bundled into two resource files that are stored as Dynamics 365 Portal web files, and the main single page application entry-point is stored as Dynamics 365 Portal web template. More information [web files](https://docs.microsoft.com/en-us/dynamics365/customer-engagement/portals/web-files) and [web templates](https://docs.microsoft.com/en-us/dynamics365/customer-engagement/portals/store-content-web-templates).
-
-Although, the frontend definition is hosted on Dynamics 365 Portal, you can still fully customize it. The backend part of the web application is not customizable since the **REST API** endpoints are hosted on Dynamics 365 Portal as a set of [web pages](https://docs.microsoft.com/en-us/dynamics365/customer-engagement/portals/web-page), and the code containing the business logic behind it resides in Event Management plugins which is not customizable.
-
-![Portal Hosted](../media/portal-hosted.png "Portal Hosted")
+- [Self-hosted](#self-hosted)
 
-### Developing and customizing the web application on Dynamics 365 Portal hosted backend
-
-To develop and customize the frontend part of the web application:
+> [!NOTE]
+> For extensive customization of the frontend you need to be familiar with the [Angular framework](https://angular.io/guide/quickstart).
 
-1. Download the [Source code](https://go.microsoft.com/fwlink/?linkid=2020107).
+## Prerequisites
+To get started with customizing the event portal you need to follow this steps:
```

Comment:
State that it's applicable only if you want to customize ng-app, otherwise it can be done in any other language as our API doesn't depend on it